cluster/gce/config-common.sh

```diff
@@ -167,3 +167,17 @@ export CSI_PROXY_STORAGE_PATH="https://storage.googleapis.com/gke-release/csi-pr
 export CSI_PROXY_VERSION="${CSI_PROXY_VERSION:-v1.1.1-gke.0}"
 # csi-proxy additional flags, there are additional flags that cannot be unset in k8s-node-setup.psm1
 export CSI_PROXY_FLAGS="${CSI_PROXY_FLAGS:-}"
+# Storage path for auth-provider-gcp binaries
+export AUTH_PROVIDER_GCP_STORAGE_PATH="${AUTH_PROVIDER_GCP_STORAGE_PATH:-https://storage.googleapis.com/gke-release/auth-provider-gcp}"
```

Comment:
TODO: can we please use public OSS releases for this project if it's cloud-provider-gcp? Prefer not to be depending on endpoints the project can't fix.


xref https://github.com/kubernetes/cloud-provider-gcp/issues/433

Reply:
+1 to do this is follow up.

Reply:
I've been following up with @aojea and @ndixita to help towards getting postsubmit builds and upstream hosted releases, I think we already have binaries in a staging location.